docs/readme.md

```diff
@@ -4,6 +4,8 @@ title: FAST-DNA
 sidebar_label: What is FAST-DNA
 ---
 
+# FAST-DNA
```

Comment:
I'm a little confused as to where this is linked to, currently we seem to be targetting the files in the `en` folder.

Reply:
It is not linked at the moment, but wanted to correct the file anyway so it is valid (I was already there). This file will probably turn into a top level readme for the docs section but that is not part of this PR.